projects/discussion-ui/src/lib/components/lib-entry/lib-entry.component.ts

```diff
@@ -43,4 +49,11 @@ export class LibEntryComponent implements OnInit {
     this.location.back();
   }
 
+  close(event) {
+    const eventAction = {
+      action: 'close'
```

Comment:
Make this string as unique name "DF_CLOSE"

Reply:
Done